app/Busline.php

```diff
@@ -19,18 +18,9 @@
  * @property Carbon|null          $updated_at
  * @property Collection|Station[] $stations
```

Comment:
`$stations` property should be gone, right?

+ there is still `group` method in this model, which i don't think is being used